File: Kattis/CPP/IntroProblems/tarifa.cc
/*
kattis tarifa problem
Author: Agis Daniels
Solve a scan as input comes in to get data build up
NOTE 
*/

#include <bits/stdc++.h>

using namespace std;

int main()
{
    //declare variables
    int x=0, n=0, ans=0, pi=0;
    
    //read in inputs and solve 
    cin>>x>>n;
    while(n--){
        cin>>pi;
        ans=ans-pi+x;
    }
    
    //print answer
    cout<<ans+x<<endl;
    return 0;
}

File: Kattis/Python/IntroProblems/tarifa.py
""" 
kattis tarifa problem
Author: Agis Daniels
Solve 
NOTE a scan as input comes in to get data build up
"""
from sys import stdin as rf
def main():
    x=int(rf.readline().strip())
    n=int(rf.readline().strip())
    ans=x
    for line in rf:
        ans=ans-int(line.strip())+x
    print(ans)
if __name__=='__main__':
    main()
